recipes/datasets/plugins/plugin_consensus.py

```diff
@@ -0,0 +1,63 @@
+from kili.plugins import PluginCore
+from kili.types import Label
+
+
+class PluginHandler(PluginCore):
+    """
+    Custom plugin instance
+    """
+
+    def aggregate_object_detection_label(self, object_job_key, labels_array):
+
+        json_response = {}
+
+        for key in object_job_key:
+            job_annotation = []
+            for label in labels_array:
+                if label["jsonResponse"][key]:
```

Comment:
in python it's not the same as in js: 
<img width="2516" alt="image" src="https://user-images.githubusercontent.com/99664787/216088267-06bc47cb-3a1e-4553-a9a0-348afd1e81ad.png">
<img width="2516" alt="image" src="https://user-images.githubusercontent.com/99664787/216088327-a9703621-48d6-4cd7-bab3-06fdea34e4de.png">

Reply:
it's weird I don't get any errors

Reply:
Maybe it's a dictionnary thing idk